x-pack/legacy/plugins/ml/server/routes/README.md

```diff
@@ -0,0 +1,16 @@
+# ML Kibana API routes
+
+This folder contains ML API routes in Kibana.
+
+Each route handler requires [apiDoc](https://github.com/apidoc/apidoc) annotations in order 
+to generate documentation.
+Besides [apidoc-markdown](https://github.com/rigwild/apidoc-markdown) package is required in order to generate the markdown.
+
+For now the process is pretty manual. You need to make sure the packages mentioned above are installed globally 
+to execute the following commands from the directory this README file is located.
```

Comment:
Nit, `command` rather than `commands` and `from the directory in which this README file is located.`

Reply:
https://github.com/elastic/kibana/pull/56006/commits/fc9297f3498431f41f91440b5c9ffa664401a3c5